store/tikv/region_request.go

```diff
@@ -101,11 +112,18 @@ func (s *RegionRequestSender) SendReqCtx(bo *Backoffer, req *tikvrpc.Request, re
 		replicaRead = kv.ReplicaReadLeader
 	}
 	for {
-		ctx, err := s.regionCache.GetRPCContext(bo, regionID, replicaRead, req.ReplicaReadSeed)
+		switch storeType {
+		case TiKV:
+			rpcCtx, err = s.regionCache.GetRPCContext(bo, regionID, replicaRead, req.ReplicaReadSeed)
```

Comment:
how about using the following two namings:
* `GetRPCCtx4TiKV`
* `GetRPCCtx4TiFlash`